src/main/java/core/basesyntax/dao/DaoDriver.java

```diff
@@ -2,5 +2,5 @@
 
 import core.basesyntax.model.Driver;
 
-public interface DaoDriver extends DaoGenericInterface<Driver> {
+public interface DaoDriver extends DaoGenericInterface<Driver, Long> {
```

Comment:
```suggestion
public interface DaoDriver extends GenericDao<Driver, Long> {
```
This is required by java naming convention to add an `Interface` word.